testcases/open_posix_testsuite/conformance/interfaces/aio_cancel/5-1.c

```diff
@@ -137,8 +142,35 @@ int main(void)
 		}
 	} while (in_progress);
 
-	if (!check_one)
-		return PTS_UNRESOLVED;
+	if (!check_one_done) {
+		return NO_FINISHED;
+	} else if (!check_one_canceled) {
+		return NO_CANCELED;
+	}
+
 
 	return PTS_PASS;
 }
+
+int main(void)
+{
+	int num_retries;
+
+	if (sysconf(_SC_ASYNCHRONOUS_IO) < 200112L)
+		return PTS_UNSUPPORTED;
+
+	for (num_retries = 0; num_retries < MAX_RETRIES; ++num_retries) {
+		int ret = do_test();
+
+		if (ret == NO_FINISHED || ret == NO_CANCELED) {
+			continue;
+		}
+
+		if (ret == PTS_PASS) {
+			printf(TNAME " PASS (%d)\n", num_retries);
+		}
```

Comment:
Curly braces are not needed for single line statements.

Reply:
Is it ltp style to not use curly braces? I'll change it